booking-app/src/main/java/org/motechproject/bookingapp/domain/VisitBookingDetails.java

```diff
@@ -43,6 +45,36 @@ public VisitBookingDetails(Visit visit, LocalDate bookingActualDate) {
         this.bookingActualDate = bookingActualDate;
     }
 
+    public VisitBookingDetailsDto toDto() {
```

Comment:
I don't think this logic should be added here, because this Dto should be something like PrimeVaccinationScheduleDto